pkl/builtins/ghalint_workflow.pkl

```diff
@@ -0,0 +1,46 @@
+import "../Builtins.pkl"
+import "../Config.pkl"
+import "./test/helpers.pkl"
+
+@Builtins.meta {
+  category = "Infrastructure"
+  description = "GitHub Actions workflow linter"
+}
+const ghalint_workflow = new Config.Step {
+  glob = List(".github/workflows/*")
+  types = List("yaml")
+  check = "ghalint run"
```

Comment:
[`ghalint run` command does not support file specification.](https://github.com/suzuki-shunsuke/ghalint)